parsers/NED.py

```diff
@@ -240,10 +272,6 @@ def fetch_production_forecast(
     NED_data = format_data(json_data, logger, forecast=True)
     ENTSOE_data = _get_entsoe_forecast_data(zone_key, session, target_datetime, logger)
 
-    combined_data = ProductionBreakdownList.update_production_breakdowns(
-        production_breakdowns=ENTSOE_data,
-        new_production_breakdowns=NED_data,
-        logger=logger,
-        matching_timestamps_only=True,
-    )
+    combined_data = combine_production_data(ENTSOE_data, NED_data, logger)
```

Comment:
By adding this to the forecasts the forecasts are now always 0 unknown as we only get forecasted wind and solar values.

This could be a problem if we decide to rely more on forecast values in the future for our estimation methods. I'd rather not pollute the database with static 0 values that are almost guaranteed to be wrong.

This could work if NED.nl also forecasts unknown (other) data but I don't think they do.

Reply:
Sure, I reverted to the old method for forecasts. I have no idea if this will be accurate, but we don't really use these for anything atm.

Reply:
It should be, the only thing these are used for is the new estimation model but that is only used for Germany at the moment.